### DISCUSSION TITLE
No routes but lease obtained

### DISCUSSION THREAD

**Discussion by lburkels** [fabianishere/udm-iptv#109] [Q&A]
Hi,

Since this afternoon I have a problem. (transferred from XS4all to KPN last night. No changes made but the inet link and routes are not processed. Any thoughts?

=== Configuration ===
WAN Interface: eth8
WAN VLAN: 4 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 213.75.0.0/16  217.166.0.0/16
LAN Interfaces: br10
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: true
=== IP Link and Route ===
=== Service Logs ===
Apr 14 20:43:35 ubnt systemd[1]: Started IPTV support for the UniFi Dream Machine.
Apr 14 20:43:35 ubnt udm-iptv[11466]: Device iptv already exists.. Deleting device
Apr 14 20:43:35 ubnt udm-iptv[11466]: Obtaining IP address for VLAN interface
Apr 14 20:43:35 ubnt udm-iptv[11466]: udhcpc (v1.22.1) started
Apr 14 20:43:35 ubnt udm-iptv[11466]: Sending discover...
Apr 14 20:43:35 ubnt udm-iptv[11466]: Sending select for 10.84.62.145...
Apr 14 20:43:35 ubnt udm-iptv[11466]: Lease of 10.84.62.145 obtained, lease time 3600
Apr 14 20:43:35 ubnt udm-iptv[11466]: NATing IPTV network ranges (if necessary)
Apr 14 20:43:35 ubnt systemd[1]: udm-iptv.service: Succeeded.
Apr 14 20:43:35 ubnt systemd[1]: udm-iptv.service: Consumed 44ms CPU time.
root@ubnt:/etc#

-- ip addr show

39: iptv@eth8: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    link/ether 68:d7:9a:6b:65:85 brd ff:ff:ff:ff:ff:ff
    inet6 fe80::6ad7:9aff:fe6b:6585/64 scope link
       valid_lft forever preferred_lft forever

**Comment by fabianishere** [comment]
Did IPTV on KPN work at some point?

Also, could you check the status of the service (whether it is enabled and running)? `systemctl status udm-iptv`

**Reply by lburkels** [reply]
Actually I am in doubt if the tv worked this morning. I am not sure if my son watched live tv or just netflix... Somehow I have the feeling it's a matter of waiting..

● udm-iptv.service - IPTV support for the UniFi Dream Machine
   Loaded: loaded (/lib/systemd/system/udm-iptv.service; enabled; vendor preset:
 enabled)
   Active: inactive (dead) since Thu 2022-04-14 21:27:04 CEST; 1min 59s ago
  Process: 31636 ExecStart=/usr/lib/udm-iptv/udm-iptv start (code=exited, status
=0/SUCCESS)
  Process: 31662 ExecStop=/usr/lib/udm-iptv/udm-iptv stop (code=exited, status=0
/SUCCESS)
 Main PID: 31636 (code=exited, status=0/SUCCESS)
      CPU: 46ms

Apr 14 21:27:03 ubnt systemd[1]: Started IPTV support for the UniFi Dream Machin
e.
Apr 14 21:27:03 ubnt udm-iptv[31636]: Device iptv already exists.. Deleting devi
ce
Apr 14 21:27:03 ubnt udm-iptv[31636]: Obtaining IP address for VLAN interface
Apr 14 21:27:03 ubnt udm-iptv[31636]: udhcpc (v1.22.1) started
Apr 14 21:27:03 ubnt udm-iptv[31636]: Sending discover...
Apr 14 21:27:03 ubnt udm-iptv[31636]: Sending select for 10.84.62.145...
Apr 14 21:27:04 ubnt udm-iptv[31636]: Lease of 10.84.62.145 obtained, lease time
 3600
Apr 14 21:27:04 ubnt udm-iptv[31636]: NATing IPTV network ranges (if necessary)
Apr 14 21:27:04 ubnt systemd[1]: udm-iptv.service: Succeeded.
Apr 14 21:27:04 ubnt systemd[1]: udm-iptv.service: Consumed 46ms CPU time.

**Reply by fabianishere** [reply]
Could you try restarting? `systemctl restart udm-iptv`

**Reply by lburkels** [reply]
yes, done that already a few times. Reinstalled, purged etc. I think the smartest thing is to dig up the experia box and check if it's working that way. Keep you posted.

**Reply by fabianishere** [reply]
@lburkels Your issue seems to be related to #111, which should be fixed now with the release of v2.1.3. Could you check if the new version works for you?

**Reply by lburkels** [reply]
yes! Indeed ;-) 

=== IP Link and Route ===
49: iptv@eth8: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    inet 10.84.59.172/20 brd 10.84.63.255 scope global iptv
       valid_lft forever preferred_lft forever
10.84.48.0/20 proto kernel scope link src 10.84.59.172
213.75.112.0/21 via 10.84.48.1 metric 249

Thanks!
